src/main/java/sigma/internship/petProject/entity/Game.java

```diff
@@ -0,0 +1,46 @@
+package sigma.internship.petProject.entity;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.persistence.*;
```

Comment:
It is recommended to not use `.*` wildcard imports. Several reasons from stackoverflow:
- Because it puts a lot of stuff into your namespace (might shadow some other object from previous import and you won't know about it).
- Because you don't know exactly what is imported and can't easily find from which module a certain thing was imported (readability).

Specify only specific imports, which you use in your project.